src/analyzer/plugins/analyzerqueenmarybeats.cpp

```diff
@@ -39,11 +38,14 @@ AnalyzerQueenMaryBeats::~AnalyzerQueenMaryBeats() {
 bool AnalyzerQueenMaryBeats::initialize(int samplerate) {
     m_detectionResults.clear();
     m_iSampleRate = samplerate;
+    m_stepSize = int(m_iSampleRate * kStepSecs + 0.0001);
+    m_windowSize = m_stepSize * 2;
```

Comment:
The multiplication by 2 is not justified anywhere. The original code mentions why.

Reply:
The original code does not mention it, the original comment is " I think this is not necessarily going to be a power of two, and the host might have a problem with that, but I'm not sure we can do much about it here"
And IHMO it's pretty useless so I did not copy it.
Is a comment indicating the the window size should be the double of step size enough? Or should it go in the theory behind it or point a link to it?

Reply:
Yes, it does: "// These are the preferred window/step sizes from the BeatTrack VAMP plugin."

Reply:
...referring to the 512/1024 constants.

Reply:
done